classes/03class/exercises/c03-docker06/mfreitassm/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.7
+
+ENTRYPOINT [ "echo", "Hello" ]
```

Comment:
Great work! 
You could also use CMD to give a default value to print as an additional step.
Something like ``` CMD ["World"]``` to print ```Hello World``` if you don't provide your name during the docker run.

Reply:
Cool!! Thanks  @jayanath